### PR TITLE
Disable telemetry for the DB migration service

### PR DESCRIPTION
**Pull request by tczekajlo**
Disable telemetry for the DB migration service if the Enterprise version is used.

### CODE REVIEW

**Comment by tczekajlo** [comment]
@wochinge One small thing, telemetry for the DB migration service. Can you 👀?  🙇

File: charts/rasa-x/templates/db-migration-service-deployment.yaml [hunk 1]
```diff
@@ -61,6 +61,10 @@ spec:
 {{ toYaml .Values.dbMigrationService.extraVolumeMounts | indent 8 }}
 {{- end }}
         env:
+        {{- if contains "rasa-x-ee" .Values.dbMigrationService.name }}
+        - name: "RASA_TELEMETRY_ENABLED"
```
Comment (wochinge) [inline charts/rasa-x/templates/db-migration-service-deployment.yaml:65]: Can we always disable it? We don't track anything in there and I don't we should start tracking in the database migration container.

File: charts/rasa-x/templates/db-migration-service-deployment.yaml [hunk 2]
```diff
@@ -61,6 +61,10 @@ spec:
...
+          value: "false"
+        {{ end }}
```
Comment (wochinge) [inline charts/rasa-x/templates/db-migration-service-deployment.yaml:67]: ?
```suggestion
        {{- end }}
```

**Review by wochinge — APPROVED** [review]
🚀

File: charts/rasa-x/templates/db-migration-service-deployment.yaml (hunk 1 above)
Comment (tczekajlo) [reply charts/rasa-x/templates/db-migration-service-deployment.yaml:65]: Hmm, good point. I took it from the other part helm chart, but I think you're right and we can just disable it for the DB migration service.

**Pull request merged**